src/components/ui/Animation.js

```diff
@@ -275,12 +273,9 @@ CIQ.Animation = function (stx, animationParameters, easeMachine) {
             if (this.chart.yAxis.left > x &&
                 this.chart.yAxis.top <= y &&
                 this.chart.yAxis.bottom >= y) {
-                if (flashingColorIndex >= flashingColors.length) { flashingColorIndex = 0; }
-                context.beginPath();
-                context.moveTo(x, y);
-                context.arc(x, y, 2 + flashingColorIndex * 1.07, 0, Math.PI * 2, false);
-                context.fillStyle = flashingColors[flashingColorIndex];
-                context.fill();
+                currentSpotElement.style.top = `${y - 3}px`;
```

Comment:
One more thing: can you make `y` an integer? We don't want floating point values in `top`:

`let y = this.pixelFromPrice(price, panel) | 0;`